reactapp/src/redux/actions.js

```diff
@@ -0,0 +1,27 @@
+export function ADD_AUTH(id, email, name) {
+    return {
+        type: 'ADD_AUTH',
+        id,
+        email,
+        name
+    };
+}
+
+export function toAPI(store, action, conf) {
+    const state = store.getState()
+    fetch('http://' + state.process.API.host + ':' + state.process.API.port + '/api' + conf.url, {
+        method: 'POST',
+        headers: {
+            'Content-Type': 'application/json'
+        },
+        body: JSON.stringify(conf.data)
+    })
+        .then(res => res.json())
+        .then(result => {
+            store.dispatch({
+                ...action,
+                ...result
+            });
+        })
+        .catch(err => console.log(err))
```

Comment:
You thoughts about this are correct, ok, but
probably here should be smth like:
```
store.dispatch({type: action + '_ERROR', message: err});
```
?